plume/src/spin/Spin.sol

```diff
@@ -208,22 +213,29 @@ contract Spin is Initializable, AccessControlUpgradeable, UUPSUpgradeable, Pausa
         UserData storage _userData = $.userData[user];
 
         if (keccak256(abi.encodePacked(rewardCategory)) == keccak256(abi.encodePacked("Jackpot"))) {
-            require(block.timestamp >= $.lastJackpotClaim + 7 days, "Jackpot cooldown active");
-            //TODO: Add case for ot enough streak count
-            require(
-                _userData.streakCount >= (block.timestamp - $.campaignStartDate) / 7 days + 2,
-                "Not enough streak for jackpot"
-            );
+            // require(block.timestamp >= $.lastJackpotClaim + 7 days, "Jackpot cooldown active");
```

Comment:
ik you commented this out on purpose for testing purposes. let's uncomment this before we merge and deploy
unless @dsldsl wants to run his bot to test this stuff live?

Reply:
if we want to keep the ability to bypass the jackpot cooldown alive, maybe we can make a simple featureflag or conditional that checks:
1. for whitelisted id's (or admin role?)
2. that the contract is deployed only on testnet, not mainnet (if that's possible?)

and add a comment describing what's going on there